src/styles/blocks/form-field--small.css

```diff
@@ -0,0 +1,26 @@
+.form-field__second {
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  background-image: url(/src/images/Search.svg);
+  background-size: 11px 11px;
+  background-repeat: no-repeat;
+  background-position: 13px center;
+  padding-left: 33px;
+  height: 41px;
+  width: 100vw;
```

Comment:
Using `width: 100vw` can cause horizontal scrolling issues on smaller screens due to the scrollbar width. Consider using `width: 100%` instead to ensure the input field fits within its container.